mix.exs

```diff
@@ -20,9 +20,9 @@ defmodule EctoStateMachine.Mixfile do
    ]
   end
 
-  defp elixirc_paths(:test), do: elixirc_paths ++ ["test/support", "test/dummy"]
-  defp elixirc_paths(_), do: elixirc_paths
-  defp elixirc_paths, do: ["lib"]
+  defp elixirc_paths(:test), do: elixirc_paths ++ ["test/support"]
```

Comment:
Removing of `test/dummy` is pointless in the terms of this PR. Please remove this change and I merge this PR